src/SignalRServiceExtension/Client/AzureSignalRClient.cs

```diff
@@ -108,6 +114,34 @@ public async Task RemoveUserFromGroup(string hubName, string userId, string grou
             await serviceHubContext.UserGroups.RemoveFromGroupAsync(userId, groupName);
         }
 
+        public async Task AddConnectionToGroup(string hubName, string connectionId, string groupName)
+        {
+            if (string.IsNullOrEmpty(connectionId))
+            {
+                throw new ArgumentException($"{nameof(connectionId)} cannot be null or empty");
+            }
+            if (string.IsNullOrEmpty(groupName))
+            {
+                throw new ArgumentException($"{nameof(groupName)} cannot be null or empty");
+            }
+            var serviceHubContext = await serviceHubContextStore.GetOrAddAsync(hubName);
+            await serviceHubContext.Groups.AddToGroupAsync(connectionId, groupName);
+        }
+
+        public async Task RemoveConnectionFromGroup(string hubName, string connectionId, string groupName)
+        {
+            if (string.IsNullOrEmpty(connectionId))
+            {
+                throw new ArgumentException($"{nameof(connectionId)} cannot be null or empty");
+            }
+            if (string.IsNullOrEmpty(groupName))
+            {
+                throw new ArgumentException($"{nameof(groupName)} cannot be null or empty");
+            }
+            var serviceHubContext = await serviceHubContextStore.GetOrAddAsync(hubName);
+            await serviceHubContext.Groups.RemoveFromGroupAsync(connectionId, groupName);
```

Comment:
In Management SDK, only Persistent mode supports it for now.

Reply:
I will update Management SDK first